app/components/views/GetStartedPage/index.js

```diff
@@ -31,6 +32,11 @@ class GetStartedPage extends React.Component {
   }
 
   render() {
+    const skipForm = getMustOpenForm() === false && getAppdataPath().length>0;
+
+    if(skipForm) {
```

Comment:
You need to move this check up into `componentWillMount` above.  `render()` causes multiple incorrect requests to onStartDaemon.

Also you should put skipForm into the selectors:  feel free to diff my branch for the changes https://github.com/alexlyp/decrediton/tree/ayp_startup_fix